### DISCUSSION TITLE
wallet connect v2 with Phantom

### DISCUSSION THREAD

**Discussion by ryuk8000** [WalletConnect/walletconnect-monorepo#3067] [Developer Support]
Doesn't wallet connect v2 support Phantom as i have extension enabled in desktop for Phantom with other  wallets disabled. I am not getting Phantom wallet. Same case in mobile if Phantom is installed in mobile

**Comment by crypblizz8** [comment]
Hey @ryuk8000 can you show the modal you tried to connect with?

**Reply by jupiterrosen** [reply]
I have the same problem: 
```

const supportedWallet = ['74f8092562bd79675e276d8b2062a83601a4106d30202f2d509195e30e19673d','a797aa35c0fadbfc1a53e7f675162ed5226968b44a19ee3d24385c64d1d3c393','fd20dc426fb37566d803205b19bbc1d4096b248ac04548e3cfb6b3a38bd033aa']

<WalletConnectModal projectId={projectId} providerMetadata={providerMetadata} explorerRecommendedWalletIds={supportedWallet}/>

```
The explorerRecommendedWalletIds includes the id of the phantom wallet from Walletconnect wallet explorer
but finally I only got spot:
![6084dfc760d4ec65ddc92cb102eed50](https://github.com/WalletConnect/walletconnect-monorepo/assets/112633538/2ef4cc1e-672a-41ce-b0f8-2b561c903038)
